config/application.rb

```diff
@@ -34,29 +60,26 @@ class Application < Rails::Application
     # we do here instead of using initializers because of a Rails 5.1 vs
     # MySQL bug where `rake db:reset` causes ActiveRecord to be loaded
     # before initializers and causes configuration not to be respected.
-    # This is fixed in Rails 5.2
-    config.load_defaults 5.1
+    config.load_defaults 6.0
     config.active_record.belongs_to_required_by_default = false
     config.active_record.include_root_in_json = true
     # Make `form_with` generate non-remote forms. Defaults true in Rails 5.1 to 6.0
+    config.action_view.form_with_generates_remote_forms = false
+    # Make Ruby preserve the timezone of the receiver when calling `to_time`.
+    config.active_support.to_time_preserves_timezone = false
+    config.action_dispatch.use_authenticated_cookie_encryption = false
+    config.active_support.use_authenticated_message_encryption = false
+
+    # Use a modern approved hashing function
+    config.active_support.hash_digest_class = OpenSSL::Digest::SHA256
```

Comment:
We can leave a note that this is the default in Rails 7.0, so can be removed when we upgrade.